code-examples/php/usage-examples/bulkWrite.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+require 'vendor/autoload.php'; 
+
+// Replace the uri string with your MongoDB deployment's connection string.
+$uri = "mongodb://localhost:27017";
+
+$collection = (new MongoDB\Client($uri))->sample_mflix->movies;
+
+try {
+    $result = $collection->bulkWrite([
+        [ 'insertOne' => [[
+                'location' => [
+                    'address' => [
+                        'street1' => '3 Main St.',
+                        'city' => 'Anchorage',
+                        'state' => 'AK',
+                        'zipcode' => '99501',
+                    ]
+                ]
+            ]]
+        ],
+        [ 'insertOne' => [[
+                'location' => [
+                    'address' => [
+                        'street1' => '75 Penn Plaza',
+                        'city' => 'New York',
+                        'state' => 'NY',
+                        'zipcode' => '10001',
+                    ]
+                ]
+            ]]
+        ],
+        [ 'deleteOne' => [[ 'location.address.street1' => '221b Baker St' ]]]
+      ],
+    );
+
+    print_r("Inserted: " . $result->getInsertedCount() . "\n");
+    print_r("Removed: " .  $result->getDeletedCount() . "\n");
+    //var_dump($result);
```

Comment:
I don't think you want a comment in the final example.

Reply:
Sorry, I think I accidentally left this in when testing the code example.